etna/transforms/utils.py

```diff
@@ -0,0 +1,6 @@
+from typing import Set
+
+
+def match_target_quantiles(features: Set[str]) -> Set[str]:
+    """Find quantiles in dataframe columns."""
+    return set(i for i in list(features) if "target_" in i)
```

Comment:
can we use any regexp to avoid matching with `target_log` (result of `LogTransform` with out_column)?